### ISSUE TITLE
Responsiveness

### ISSUE THREAD

**Issue by jiurate** [limzykenneth/ualfutures-frontend#26]
[320x568.zip](https://github.com/limzykenneth/ualfutures-frontend/files/660389/320x568.zip)

**Comment by limzykenneth** [comment]
@jiurate for the version with guides did you include some text? It's just showing up as a bunch of red rectangles for me...

**Comment by limzykenneth** [comment]
Also for the header menu do you want the button on the right that bring up the full menu to use the burger icon? It's more conventional with that.

**Comment by limzykenneth** [comment]
Deployed, might have some bugs. Let me know here.

**Comment by jiurate** [comment]
Apart from the menu issue and other things that I posted elsewhere, a couple things on mobile:

1. BW filter didn't show up when I checked earlier, now it shows the blue filter on both surge and .studio addresses, although on mac I can see bw on surge.

2. Tiny tiny detail, instead of straight line this is how it shows on search bar:
![img_9592](https://cloud.githubusercontent.com/assets/13275142/22761294/073102ba-ee52-11e6-9e81-f038e282e34d.jpg)
